From: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Subject: MAINTAINERS: remove an obvious typo in MAPLE TREE
Date: Tue, 7 Jun 2022 08:38:34 +0200

Commit c6e2362e9289 ("Maple Tree: add new data structure") adds the maple
tree and a corresponding MAINTAINERS section, but slips in an obvious
typo.

Hence, ./scripts/get_maintainer.pl --self-test=patterns complains about a
broken reference.

Remove this obvious typo in MAPLE TREE.

Link: https://lkml.kernel.org/r/20220607063834.7004-1-lukas.bulwahn@gmail.com
Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
Acked-by: Liam R. Howlett <Liam.Howlett@Oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/MAINTAINERS~maple-tree-add-new-data-structure-fix-5
+++ a/MAINTAINERS
@@ -11800,7 +11800,7 @@ F:	include/linux/maple_tree.h
 F:	include/trace/events/maple_tree.h
 F:	lib/maple_tree.c
 F:	lib/test_maple_tree.c
-F:	tools/testing/_adix-tree/linux/maple_tree.h
+F:	tools/testing/radix-tree/linux/maple_tree.h
 F:	tools/testing/radix-tree/maple.c
 
 MARDUK (CREATOR CI40) DEVICE TREE SUPPORT
_
